### ISSUE TITLE
Single web interface for multiple OliveTin instances

### ISSUE THREAD

**Issue by jamesread** [OliveTin/OliveTin#35]
**Is your feature request related to a problem? Please describe.**
Some people want to have OliveTin instances running on multiple servers, and don't want to (or cannot) rely on SSH for remote command execution. Rather than have to remember the URLs for multiple web interfaces, it would be nice to have all actions displayed in a single web interface. 

**Design A:** Master Server to (Client) Server execution - this would break things like authentication, and would mean that the server would have to implement it's own client - which is a bit funky. 

**Design B:** Master Server advertises client servers. The webui could then connect to multiple servers, and then display results elegantly in the UI. This has the advantage of keeping the webui and server code very clean, and doens't break client auth via a LB. 

**Describe the solution you'd like**
All buttons should be in a single webui. 

Would depend on #16 first
